src/sgame/sg_bot_nav.cpp

```diff
@@ -697,15 +703,6 @@ void BotMoveToGoal( gentity_t *self )
 	switch ( self->client->pers.team )
 	{
 		case TEAM_HUMANS:
-			//dont sprint or dodge if we dont have enough stamina and are about to slow
-			if ( self->client->ps.stats[ STAT_STAMINA ] < staminaJumpCost )
-			{
-				usercmdReleaseButton( botCmdBuffer.buttons, BUTTON_SPRINT );
-
-				// walk to regain stamina
-				BotWalk( self, true );
-			}
-
 			// except if target is friendly, then let's reach it quickly
```

Comment:
comment here doesn't make sense now

Reply:
The «moonwalk bug» is something you can only see on bots, when they are out of stamina: they constantly switch between walk/run modes, resulting in them moving slower than if they were walking.
The bug is named after how the BSuit users look like, I don't remember who named it.

It already exists in codebase, but can't be seen because currently bots only run when in fight, and the moves are then too chaotic to notice this easily.